src/main/resources/logback-spring.xml

```diff
@@ -39,7 +39,7 @@
                 <provider class="net.logstash.logback.composite.loggingevent.LoggingEventPatternJsonProvider">
                     <pattern>
                         {
-                        "message": "%replace(%msg){'([A-Z]{2}[0-9]{3})[0-9]{3}[A-D]','$1****'}"
+                        "message": "%replace(%msg){'([A-Za-z]\\s*[A-Za-z]\\s*[0-9]\\s*[0-9]\\s*[0-9]\\s*)[0-9]\\s*[0-9]\\s*[0-9]\\s*[A-Da-d]','$1****'}"
```

Comment:
`/(\w{2}\d)\d{5}[A-Da-d]/`?
Or if whitespace is important: `((\w\s*){2}\d\s*)(\d\s*){5}[A-Da-d]`

Reply:
Does \w include 0-9?  They are omitted on purpose (this is a nino).  However I think the whitespace change is better.